frontends/mit-open/src/pages/DashboardPage/SettingsPage.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react"
+import {
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+  Link,
+  styled,
+} from "ol-components"
+import { useUserMe } from "api/hooks/user"
+import {
+  useSearchSubscriptionDelete,
+  useSearchSubscriptionList,
+} from "api/hooks/searchSubscription"
+
+const SOURCE_LABEL_DISPLAY = {
+  topic: "Topic",
+  unit: "MIT Unit",
+  department: "MIT Academic Department",
+  "saved search": "Saved Search",
+}
+
+const FollowList = styled(List)(() => ({
+  borderRadius: "4px",
+  background: "#fff",
+  padding: "0px",
+  margin: "0px",
+}))
+
+const StyledLink = styled(Link)(({ theme }) => ({
+  color: theme.custom.colors.red,
+}))
+
+const TitleText = styled(Typography)(({ theme }) => ({
+  marginTop: "20px",
+  marginBottom: "5px",
+
+  color: theme.custom.colors.darkGray2,
+  ...theme.typography.subtitle1,
```

Comment:
```suggestion
  ...theme.typography.h5,
```
<img width="779" alt="Screenshot 2024-07-30 at 3 39 51 PM" src="https://github.com/user-attachments/assets/9777def5-221b-435b-8acc-f1a6863492a9">